### PR TITLE
Cross-chain fork tests

### PR DESCRIPTION
**Pull request by hieronx**

### CODE REVIEW

File: .github/workflows/ci-fork.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: "CI Fork"
+
+on:
+  push:
+    branches:
+      - main
+  schedule:
+    - cron: "0 13 * * 1"
```
Comment (hieronx) [inline .github/workflows/ci-fork.yml:8]: Runs automatically once per week, on Monday

File: .github/workflows/ci-deep.yml [hunk 2]
```diff
@@ -45,10 +45,6 @@ jobs:
         run: forge test --no-match-path "test/invariants/**/*.sol"
         env:
           FOUNDRY_PROFILE: ci
-          CENTRIFUGE_CHAIN_ORIGIN: ${{ vars.CENTRIFUGE_CHAIN_ORIGIN }}
-          MAINNET_RPC_URL: ${{ vars.MAINNET_RPC_URL }}
```
Comment (hieronx) [inline .github/workflows/ci-deep.yml:49]: Replaced by the foundry config

File: .github/workflows/ci-deep.yml [hunk 3]
```diff
@@ -45,10 +45,6 @@ jobs:
         run: forge test --no-match-path "test/invariants/**/*.sol"
         env:
           FOUNDRY_PROFILE: ci
-          CENTRIFUGE_CHAIN_ORIGIN: ${{ vars.CENTRIFUGE_CHAIN_ORIGIN }}
```
Comment (hieronx) [inline .github/workflows/ci-deep.yml:48]: Anyways not used anymore

File: .github/workflows/ci-deep.yml [hunk 4]
```diff
@@ -45,10 +45,6 @@ jobs:
...
-          MAINNET_RPC_URL: ${{ vars.MAINNET_RPC_URL }}
-          POLYGON_RPC_URL: ${{ vars.POLYGON_RPC_URL }}
-          SETUP_TEST_DATA: false
```
Comment (hieronx) [inline .github/workflows/ci-deep.yml:51]: Not used for testing anymore

**Review by ilinzweilin — APPROVED** [review]

**Pull request merged**